charts/rancher-webhook/templates/pre-delete-hook-psp.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.preDelete.enabled (.Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy") }}
+{{- if .Values.global.cattle.psp.enabled  -}}
```

Comment:
I am not a Helm expert - why is there a dash before the ending braces here but not on line 19 in the `pre-delete-cluster-role.yaml`?

Reply:
Helm uses the - to control the use of whitespace. In this case, AFAIK this tells helm to "eat" the whitespace. In this case, it does not appear to make a difference testing on my local with the trailing `-` and without the trailing `-` using `helm install --dry-run rancher-webhook -n cattle-system` did not produce a difference in whitespace.

Now this does not hold for the change in the `ClusterRole.yaml`. If you include the `-` on the if statement there you get a rendering error.

Reply:
Interesting, thanks for testing this out!